src/plugins/controls/connectivity/pauseOnError.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2020 (original work) Open Assessment Technologies SA ;
+ */
+
+import __ from 'i18n';
+import pluginFactory from 'taoTests/runner/plugin';
+import dialogTpl from 'taoQtiTest/runner/plugins/controls/connectivity/pauseOnError.tpl';
+
+const name = 'pauseOnError';
+const dialogMessage = {
+    title: __('Something unexpected happened.'),
+    message: __('Please try reloading the page or pause the test. If you pause, you will be able to resume the test from this page.')
+};
+const dialogConfig = {
+    buttons: {
+        labels: {
+            ok: __('Pause the test'),
+            cancel: __('Reload the page')
+        }
+    }
+};
+const pauseMessage = __('You encountered an unexpected issue. The test has been suspended.');
+const pauseContext = {
+    reasons: {
+        category: 'technical',
+        subCategory: 'error'
+    },
+    message: pauseMessage,
+    originalMessage: 'Due to an unexpected issue the test has been suspended.'
```

Comment:
Shouldn't it be translated?

Reply:
Not this one. This is a text that will be logged.